docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '2'
+services:
+  gem:
+    image: ruby:2.4.0
+    depends_on:
+      - nyc-geoclient
+    volumes:
+      - .:/opt/nyc_geo_client
+  nyc-geoclient:
+    container_name: nyc-geoclient
+    image: 'delner/geoclient:16b'
+    expose:
+      - 8080
```

Comment:
Currently we don't have any dependency with docker, should we have docker-compose file here?

Reply:
I guess that depends. I added this file because it was useful "dev tool" (anyone can use it to drive integration tests with real data/service interaction.) I probably wouldn't recommend adding a dependency to Docker for CI (the mocks/stubs you have are fine), but if anyone wanted to fork and develop/test new features against Geoclient, you could just run this with `docker-compose up` and point the config to `localhost:8080`.